src/applications/debt-letters/reducers/index.js

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
Is this intentional as boilerplate?

Reply:
@LinnJS  Intentional - I am adding the reducer / actions in the next PR